source/enum_set.h

```diff
@@ -149,25 +271,31 @@ class EnumSet {
     return bucket.data & ComputeMaskForValue(value);
   }
 
+  // Wrapper on `Contains` which matches the STL API for sets.
+  inline size_t count(T value) const { return contains(value) ? 1 : 0; }
+
   // Calls `unaryFunction` once for each value in the set.
   // Values are sorted in increasing order using their numerical values.
+  // FIXME(Keenuts): replace usages with either ranged-for or std::for_each.
```

Comment:
This should be a TODO with an issue number.
```suggestion
  // TODO(#NNNN): replace usages with either ranged-for or std::for_each.
```